koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -340,6 +340,12 @@ def _get_repositories(self, response):
                               repo_dict, type(error), error)
         return repositories
 
+    def _get_koji_build_id(self, response):
+        if hasattr(response, "get_koji_build_id"):
+            return response.get_koji_build_id()
+        else:
+            self.logger.info("Koji content generator build ID not available.")
```

Comment:
`None` is probably best (i.e. make that explicit).

I'll note that `get_koji_build_id()` might return either `None` (that build didn't have an annotation for it) or '' (annotation was empty). It's fine, just something to be aware of.